src/main/java/com/quizapp/enterprise/services/GameService.java

```diff
@@ -75,9 +80,16 @@ private boolean userNameExists(String userName, String gameCode){
 
     }
 
-    public GuessResult ProcessPlayerGuess(String userGuess, String gameCode, Long questionId, String playerName) {
+    public GuessResult ProcessPlayerGuess(String userGuess,
+                                          String gameCode,
+                                          Long questionId,
+                                          String playerName) throws Exception {
```

Comment:
reformatted the arguments to use indentation so the code may be cleaner and easier to read.
This recommended by the "methods" section of the java best practices summary referenced below.

source: https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3